### ISSUE TITLE
TensorBoard Callback not working

### ISSUE THREAD

**Issue by AbhishekAnand18** [cyprienruffino/CTCModel#7]
Hi,

I get this below error when I use TensorBoard Callback in the network.fit() method 
**ValueError:** An operation has `None` for gradient. Please make sure that all of your ops have a gradient defined (i.e. are differentiable). Common ops without gradient: K.argmax, K.round, K.eval

But if I remove it it works fine. do you suggest any method to use TensorBoard with fit()  method?

I have set validation_split=0.1 in model.fit() method, val_loss is calculated but get the error when I use TensorBoard CallBack
